readthedocs/rtd_tests/tests/test_project_symlinks.py

```diff
@@ -58,10 +58,12 @@ def test_symlink_basic(self):
             'ln -nsf {translation}/rtd-builds {project}/translations/de',
             'ln -nsf {builds} {project}/translations/en',
         ]
-        for (i, command) in enumerate(commands):
-            self.assertEqual(self.commands[i], command.format(**self.args),
-                             msg=('Command {0} mismatch, expecting {1}'
-                                  .format(i, self.commands[i])))
+
+        for command in commands:
+            self.assertIsNotNone(
+                self.commands.pop(
+                    self.commands.index(command.format(**self.args))
+                ))
```

Comment:
Tox introduces `PYTHONHASHSEED` env variable to testing, which caught these bugs as failing due to dictionary ordering inconsistencies. This is included here to keep the testing experience the same as before the PR.